### DISCUSSION TITLE
Brownfield android theming

### DISCUSSION THREAD

**Discussion by francois-pasquier** [zoontek/react-native-edge-to-edge#41] [Q&A]
Reading the doc, I read we have to edit the AppTheme as such:
```
<resources>
- <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+ <style name="AppTheme" parent="Theme.EdgeToEdge">
    <!-- … -->
  </style>
</resources>
```

What about brownfield apps where we don't really want to touch this?
I think the doc could be improved regarding this topic

**Comment by zoontek** [comment]
@francois-pasquier Each brownfield app is unique, so there's probably not a magic config that could adapt to each one.
The theme is actually not required. It's used to avoid an initial system bars switch, that isn't visible if you are using a [splashscreen](https://github.com/zoontek/react-native-bootsplash). It also apply style on RN Modal (which is somehow a fix until [this PR](https://github.com/facebook/react-native/pull/47254) is merged).

Just make sure you remove everything [from this list](https://github.com/zoontek/react-native-edge-to-edge/blob/1.0.1/src/expo.ts#L8), except `android:navigationBarColor` / `android:statusBarColor` that could be `@android:color/transparent`.

That should do the trick.
